deepseg.cc

```diff
@@ -113,6 +113,18 @@ const std::vector<std::string> labels = { "background", "aeroplane", "bicycle",
 const size_t cnum = labels.size();
 const size_t pers = std::distance(labels.begin(), std::find(labels.begin(),labels.end(),"person"));
 
+enum class ModelType {
+	BodyPix,
+	DeepLab,
+	GoogleMeetSegmentation,
+	MLKitSelfie,
+};
```

Comment:
More consistent naming? Also following STL naming conventions snake_case is preferred.
```suggestion
enum class model_type {
	BodyPix,
	DeepLab,
	GoogleMeetSegmentation,
	MLKitSelfie,
};
```

Reply:
Should the enum values actually be lower case also if following the STL conventions?

Reply:
Good question. Strictly speaking yes, but given the current names mesh quite well from a descriptiveness PoV converting them to snake-case would probably lose some of the appeal.

Reply:
Ok lets keep it as is then 🙂